services/discord/discord.service.js

```diff
@@ -62,6 +67,11 @@ module.exports = class Discord extends BaseJsonService {
     }
   }
 
+  constructor(context, config) {
+    super(context, config)
+    this._runningOnHeroku = config.shieldsProductionHerokuHacks
```

Comment:
Should we sync up these names? e.g. use shieldsProductionHerokuHacks for the instance var?

Reply:
yeah, missed that one somehow. updating now